Source/Scene/ParticleSystem.js

```diff
@@ -817,12 +1010,33 @@ define([
     };
 
     /**
-     * A function used to apply a force to the particle on each time step.
+     * A function used to modify attributes of the particle at each time step. This can include force modifications,
+     * color, sizing, etc. See the {@link https://cesiumjs.org/tutorials/Particle-Systems-Tutorial/|Particle Systems Tutorial}
```

Comment:
Try the see tag:

`@see {@link https://cesiumjs.org/tutorials/Particle-Systems-Tutorial/|Particle Systems Tutorial}`